tests/unit_tests/tools/skill_tools/test_skill_manager_unit.py

```
@@ -0,0 +1,245 @@
+        registry.get_skill_count.return_value = 0
+        config = SkillConfig(marketplace_url="http://nonexistent:9999")
+        manager = SkillManager(config=config, registry=registry)
+        results = manager.search_marketplace(query="test")
+        assert results == []
+
+    def test_install_from_marketplace_error(self):
+        registry = MagicMock()
+        registry.get_skill_count.return_value = 0
+        config = SkillConfig(marketplace_url="http://nonexistent:9999")
+        manager = SkillManager(config=config, registry=registry)
+        ok, msg = manager.install_from_marketplace("test")
```

Ruff (0.15.2)

[warning] Unpacked variable `msg` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)